templates/apache2_nc.j2

```diff
@@ -54,6 +54,11 @@
   SSLCertificateChainFile {{ nextcloud_tls_cert_chain_file }}
 {% endif %}
 
+{% if (nextcloud_computed_major_version|int) > 13 %}
```

Comment:
IDK if it's still a need to check Nextcloud version. Version should be higher than 13 in 100% and if it's not that's asking out trouble.
BTW versions < 22 are unsupported right now: https://nextcloud.com/changelog/

Reply:
@wiktor2200 removed nextcloud < 13 code.